andela/workdey
# #[120196463] Task acceptance/rejection notification bug

#### What does this PR do?

It fixes the notification dialog box for task acceptance or task rejection
#### How should this be manually tested?
- log in as a taskee
- Click on notifications
- View the individual tasks
- Click on accept or rreject
- A dialog box should pop up with description based on accepting or rejecting a task
#### What are the relevant pivotal tracker stories?
#120196463
#### Screenshots (if appropriate)

![screen shot 2016-06-01 at 16 23 37](https://cloud.githubusercontent.com/assets/14159384/15715219/86cef314-2815-11e6-88da-447fe37bca82.png)

![screen shot 2016-06-01 at 16 23 19](https://cloud.githubusercontent.com/assets/14159384/15715195/71efd74c-2815-11e6-9860-2e8274f50206.png)

![screen shot 2016-06-01 at 16 20 09](https://cloud.githubusercontent.com/assets/14159384/15715185/6c1e1306-2815-11e6-8b2a-70cb19edbb7a.png)
